content/articles/getting-started-with-nessus-scanner/index.md

```diff
@@ -0,0 +1,218 @@
+
+If you do not have Kali-Linux installed on your pc, do not worry, we will show you how to set this up quickly.
+
+### Step 1:
+
+### Installing Kali Linux
+
+Use the link below to get to the download page:
+
+- [Kali Linux iso](https://www.kali.org/get-kali/#kali-platforms)
+
+#### Download Vmware (virtual box)
+
+After successfully downloading the Kali-Linux Iso, you can now download the VMware using the link below:
+
+- [Vmware](https://www.vmware.com/products/workstation-player/workstation-player-evaluation.html)
+
+Follow the [installing Vmware](https://forums.tomshardware.com/faq/how-to-install-vmware-workstation-in-windows-10.3273363/) tutorials to install the VMware.
+
+Start Kali-Linux after the installation and enter your username and password when prompted.
+
+**Note:** The **default** username is **root** and the default password is **toor**.
+
+### Step 2:
+
+### Download Nessus Tenable
+
+- [Nessus Tenable](https://www.tenable.com/downloads/api/v1/public/pages/nessus/downloads/14957/download?i_agree_to_tenable_license_agreement=true)
+
+After downloading Nessus Tenable, follow the steps below to install Nessus on Kali-Linux:
+
+- Start the **terminal** by:
+
+ - Holding the **cltr+alt+T** buttons simultaneously or
+   - Using the GUI (Graphical User Interface), go to the application section and double click on the terminal.
+
+Run the command below on the terminal to update and upgrade your Kali Linux.
+
+```bash
+    $ sudo apt-get upgrade && update -y
+```
+Now, let us change our directory to Downloads.
+
+```bash
+    $ cd Downloads
+```
+Use the command below to list all files in `Downloads`, then copy the downloaded file name.
+
+```bash
+    $ ls
+``` 
+Use the command below to grab the package to download and install it.
+
+```bash
+    $ dpkg -i Nessus-10.0.2-ubuntu910_amd64.deb
+```
+![dpkg](dpkg-nessus.png)
+
+Start Nessus by typing the following into your terminal.
+
+```bash
+    $ /bin/systemctl start nessusd.service
+```
+Navigate to the following URL in your browser to start Nessus.
+
+- [Nessus localhost](https://kali:8843/) to configure your scanner.
+
+- Default port to run Nessus is **`8834`**.
+
+If you see a connection not secured page, click *advance* accept risk and continue.
+
+
+Congratulations, you now have Nessus installed on Kali-Linux.
+
+Wait for the necessary Nessus plugins to compile; click `Nessus Essentials` and the continue button after the compilation.
+
+### Step 3:
+- Provide your name and a valid email to receive an activation code.
+
+- Once your activation code has arrived in your email, go ahead and copy it.
+
+- Paste the activation code into the activation box and click continue.
+
+- Provide your username and password for login
+
+- Nessus will start the initializing process, which will take a couple of minutes; let it install while you get a drink or coffee.
+
+- That took forever! All right, we have loaded Nessus, and it is installed.
+
+![Metersploitable](metersploitable.png)
```

Comment:
This formatting for referencing an image is wrong. Please refer to this [article](https://github.com/section-engineering-education/engineering-education/pull/6394/files) on line 124 to see the correct way.